File: AndroidStudioProjects/HideandShowDemo/app/src/main/java/com/wambuacooperations/hideandshowdemo/MainActivity.java
package com.wambuacooperations.hideandshowdemo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import org.w3c.dom.Text;

public class MainActivity extends AppCompatActivity {

    TextView displayTextView;
    public void show(View view){
        displayTextView.setVisibility(View.VISIBLE);
    }
    public void hide(View view){
        displayTextView.setVisibility(View.INVISIBLE);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        displayTextView=findViewById(R.id.displayTextView);
    }
}

File: KotlinFun/app/src/main/java/com/wambuacooperations/kotlinfun/MainActivity.kt
package com.wambuacooperations.kotlinfun

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var counter=0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var textView=findViewById<TextView>(R.id.textView)

        textView.text="0"

    }

    fun addOne(view: View){
        counter++
        textView.text=counter.toString()

    }
    fun reset(view:View){
        counter=0
        textView.text=counter.toString()
    }
}
